.gitignore

```diff
@@ -1,36 +1,17 @@
-.DS_Store
```

Comment:
.DS_Store should always be in your gitignore. It is a record of where the file is located within the folder when viewing via a graphical user interface. It is not a change that should be checked in. Two ways to prevent this are to firstly add it to the gitignore and secondly make sure the files you add are the changes you want to add to the repository.